docs/io.cozy.apps.md

```diff
@@ -74,3 +74,15 @@ This is the truncated version of the official Cozy `drive` app.
   }
 }
 ```
+
+# Specials `io.cozy.apps` doctypes 
```

Comment:
Why not using io.cozy.files instead of io.cozy.apps ?

Reply:
Because it more about the application that use the file than the file itself.